index.html

```diff
@@ -201,6 +201,11 @@ <h3>Sampling and Reporting Rate</h3>
       The <dfn>sampling rate</dfn> for a [=platform collector=] is defined as a rate
       at which the [=user agent=] obtains telemetry readings from the underlying platform,
       and it might differ from the pressure observers' [=requested sampling rates=].
+      The rate is measured in Hertz (cycles per second).
+    </p>
+    <p>
+      The [=requested sampling rate=] can be derived from the [=requested sample interval=]
```

Comment:
"requested sampling rate" was `<dfn>`ed in PressureObserverOptions, now it's not defined anywhere.

My suggestion was to move the "requested sampling interval"'s `<dfn>` to this section and also `<dfn>` "requested sampling rate" here, and explain how they are basically the same thing.

Reply:
Should be fixed now

Reply:
I still think the current `<dfn>` for "requested sample interval" that is currently in PressureObserverOptions should be moved to this section, so that you group all related concepts in one place.

Reply:
I am trying to follow you, so you don't want the dfn in the <h3>The <dfn>sampleInterval</dfn> member</h3> section as it is now (and the old one previously was)?

Reply:
Correct. IMO it makes more sense to put the `<dfn>`s for both "requested sampling rate/interval" in the "Sampling and Reporting Rate" section, so that both concepts are defined next to each other. The "sampleInterval member" section then just references the concept you defined in the "Concepts" section.

Reply:
OK check now!